Gems/EMotionFX/Code/Source/Editor/Plugins/Ragdoll/PhysicsSetupViewportUiCluster.cpp

```diff
@@ -8,28 +8,42 @@
 
 #include <EMotionFX/Source/Transform.h>
 #include <EMotionFX/Tools/EMotionStudio/EMStudioSDK/Source/RenderPlugin/ViewportPluginBus.h>
-#include <Editor/Plugins/Ragdoll/PhysicsSetupColliderTranslationManipulators.h>
+#include <Editor/Plugins/Ragdoll/ColliderRotationManipulators.h>
+#include <Editor/Plugins/Ragdoll/ColliderTranslationManipulators.h>
 #include <Editor/Plugins/Ragdoll/PhysicsSetupViewportUiCluster.h>
 
 namespace EMotionFX
 {
     PhysicsSetupViewportUiCluster::PhysicsSetupViewportUiCluster()
     {
-        m_subModes[SubMode::ColliderTranslation] = AZStd::make_unique<PhysicsSetupColliderTranslationManipulators>();
+        m_subModes[SubMode::ColliderTranslation] = AZStd::make_unique<ColliderTranslationManipulators>();
+        m_subModes[SubMode::ColliderRotation] = AZStd::make_unique<ColliderRotationManipulators>();
+    }
+
+    AZ::s32 PhysicsSetupViewportUiCluster::GetViewportId()
```

Comment:
suggestion: This could be made `const` and `m_viewportId` could be marked mutable as this is an internal optimization technique

Reply:
Done.